content/docs/gke/gcp-e2e.md

```diff
@@ -556,27 +508,44 @@ run it on your GKE cluster.
 
 ### Prepare your training component to run on GKE
 
-To run the TensorFlow training job on your cluster on GKE, you use the *train* 
-component (`train.jsonnet`) that you copied into your `my_ksonnet_app` directory 
-earlier in this tutorial. Prepare the component by setting the following
-parameters:
+Enter the `training/GCS` directory.
+
+```
+cd ${WORKING_DIR}/training/GCS
+```
 
-1. Use the [`ks param set`][ks-param-set] command to customize the component’s
-   parameters, pointing to your training container in Container Registry and
-   your Cloud Storage bucket:
+Give the job a different name (to distinguish it from your job which didn't use GCS)
```

Comment:
s/GCS/Cloud Storage

Please do the above for all references to Cloud Storage that are straight text, that is all references that are not part of code or a directory name. The brand name is "Cloud Storage", not GCS.